test/WebSites/FilesWebSite/Controllers/UploadFilesController.cs

```diff
@@ -21,5 +23,22 @@ public class UploadFilesController : Controller
 
             return Json(resultUser);
         }
+
+        [HttpPost("UploadProductSpecs")]
+        public IActionResult ProductSpecs(Product product)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            var files = new Dictionary<string, List<string>>();
```

Comment:
Deserves a comment about returning filenames instead of `IFormFile`s.

You could also create another `public` DTO and use that here since everything is accessible w/in `FormFileUploadTest`.